src/main/kotlin/no/nav/syfo/pdl/service/PdlPersonService.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.syfo.pdl.service
+
+import io.ktor.util.KtorExperimentalAPI
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.client.StsOidcClient
+import no.nav.syfo.log
+import no.nav.syfo.pdl.client.PdlClient
+import no.nav.syfo.pdl.model.Navn
+import no.nav.syfo.pdl.model.PdlPerson
+import no.nav.syfo.util.LoggingMeta
+
+@KtorExperimentalAPI
+class PdlPersonService(private val pdlClient: PdlClient, private val stsOidcClient: StsOidcClient) {
+
+    suspend fun getPersonnavn(fnr: String, loggingMeta: LoggingMeta): PdlPerson? {
+        val stsToken = stsOidcClient.oidcToken().access_token
+        val pdlResponse = pdlClient.getPerson(fnr, stsToken)
+        if (pdlResponse.data.hentPerson == null) {
+            log.error("Fant ikke person i PDL {}", StructuredArguments.fields(loggingMeta))
+            return null
+        }
+        if (pdlResponse.data.hentPerson.navn.isNullOrEmpty()) {
```

Comment:
Ta gjerne en avsjekk med Camilla på om det er riktig å opprette fordelingsoppgave hvis bruker finnes i PDL, men mangler navn. Tror vel egentlig ikke det er sannsynlig at det kommer til å skje, og da er det kanskje unødvendig å lage masse spesiallogikk kun for å ta høyde for at det i teorien kan skje (går evt an å høre med MFN-folkene om det er mange brukere som finnes i PDL, men som mangler navn).